operations/phlare/helm/phlare/templates/ingress.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "phlare.fullname" . }}-ingress
+  labels:
+    {{- include "phlare.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  rules:
+  {{- range $.Values.ingress.hosts }}
+  - host: {{ . | quote }}
+    http:
+      paths:
+      - backend:
+          service:
+            name: {{ include "phlare.fullname" $ }}-query-frontend
+            port:
+              number: 4100
+        path: /querier.v1.QuerierService/
+        pathType: Prefix
+      - backend:
+          service:
+            name: {{ include "phlare.fullname" $ }}-distributor
+            port:
+              number: 4100
+        path: /push.v1.PusherService/
+        pathType: Prefix
```

Comment:
This will not work when we deploy a single binary version of Phlare. I suggest you either:

* Have a if/else for that case and just use the one service
* We always create the distributor / query-frontend services but in the single binary they should just point to the same pod(s)

Reply:
I think I will go with the if else for minimal changes, if that's ok for you.

Reply:
Yes that sounds good to me, also feel free to try any other idea you might have. Those two just came to my mind thinking about it.

Reply:
Does counting the number of keys to check if it is in micro-service mode satisfying to you? It feels weird but I am not used to helm world.